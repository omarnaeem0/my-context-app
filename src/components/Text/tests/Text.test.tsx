import * as React from 'react';
import * as TestRenderer from 'react-test-renderer';
import { Text } from '../../index';

test('Should find the Text component', () => {
	const component = TestRenderer.create(
		<Text />
	);
	const tree = component.toTree();
	expect(tree).toMatchSnapshot();
});

test('Should find the Text component', () => {
	const component = TestRenderer.create(
		<Text>Hello</Text>
	);
	const tree: any = component.toJSON();
	expect(tree.children).toEqual(['Hello']);
});
