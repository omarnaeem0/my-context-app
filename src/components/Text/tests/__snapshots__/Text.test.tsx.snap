// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should find the Text component 1`] = `
Object {
  "instance": null,
  "nodeType": "component",
  "props": Object {},
  "rendered": Object {
    "instance": null,
    "nodeType": "host",
    "props": Object {
      "children": undefined,
    },
    "rendered": Array [],
    "type": "div",
  },
  "type": [Function],
}
`;

exports[`Should find the Text component 2`] = `
Object {
  "instance": null,
  "nodeType": "component",
  "props": Object {
    "children": "Hello",
  },
  "rendered": Object {
    "instance": null,
    "nodeType": "host",
    "props": Object {
      "children": "Hello",
    },
    "rendered": Array [
      "Hello",
    ],
    "type": "div",
  },
  "type": [Function],
}
`;
